src/_C014.cpp

```diff
@@ -63,7 +66,7 @@ String CPlugin_014_pubname;
 bool   CPlugin_014_mqtt_retainFlag = false;
 
 void C014_replaceSysname(String& var) {
-  var.replace(F("%sysname%"), Settings.getName());
+  var.replace(F("%sysname%"), Settings.getHostname()); // Used to be getName(), but that doesn't include the UnitNr when configured
```

Comment:
Why is this done in a separate function in C014 and not just call the generic parse function we use everywhere else in ESPEasy?  (I know the answer, because C014 is completely "written" using <copy><paste><paste><paste><paste><paste><paste><paste><paste><paste>)

And on top of that the same %sysname% should be replaced by the same as anywhere else in ESPEasy.
If something else is needed here, we should add a new system variable, not make a different implementation for this controller.